app/Models/Multiplayer/Game.php

```diff
@@ -19,24 +19,55 @@
  */
 namespace App\Models\Multiplayer;
 
+use App\Libraries\ModsFromDB;
+use App\Models\Beatmap;
+
 class Game extends Model
 {
     protected $table = 'games';
     protected $primaryKey = 'game_id';
+
     protected $hidden = ['match_id'];
+
     protected $dates = [
         'start_time',
         'end_time',
     ];
+
     public $timestamps = false;
 
+    protected $casts = [
+        'mods' => 'integer',
```

Comment:
I believe this is ignored because there's a `getModsAttribute` function defined. Also the mysqlnd driver already casts integer.